clap_mangen/tests/testsuite/roff.rs

```diff
@@ -105,3 +105,17 @@ fn value_name_without_arg() {
         cmd,
     );
 }
+
+#[test]
+fn flatten_help_false() {
+    let name = "my-app";
+    let cmd = common::basic_command(name).flatten_help(false);
+    common::assert_matches(snapbox::file!["../snapshots/basic.bash.roff"], cmd);
+}
+
+#[test]
+fn flatten_help_true() {
+    let name = "my-app";
+    let cmd = common::basic_command(name).flatten_help(true);
+    common::assert_matches(snapbox::file!["../snapshots/basic.bash.roff"], cmd);
+}
```

Comment:
Please use `flatten_help.roff` in this commit.  That way we can see how `flatten_help.roff` itself changes between commits.

Reply:
Fixed